tasks/configure.yml

```diff
@@ -1,6 +1,28 @@
 ---
+
 - name: configure rabbitmq
   template:
     src: rabbitmq.config.j2
     dest: /etc/rabbitmq/rabbitmq.config
   when: rabbitmq_config is defined
+
+- name: rabbitmq full stop
+  include: node_fullstop.yml
+  when: rabbitmq_erlang_cookie is defined
+
+- name: set erlang cookie
+  template:
+    src: erlang.cookie.j2
+    dest: "{{ rabbitmq_erlang_cookie_file }}"
+    owner: rabbitmq
+    group: rabbitmq
+    mode: 0400
+  when: rabbitmq_erlang_cookie is defined
+
+- include: node_fullstart.yml
+
+- include: vhosts.yml
+
+- include: users.yml
```

Comment:
As I noted in #6, this will fail when creating new users (at least it did for me). It would be good to only run this on the master node.